app/population/view/dashboard.njk

```diff
@@ -0,0 +1,32 @@
+{% extends "layouts/collection.njk" %}
+
+{#{% block pageTitle %}#}
+{#  {{ t("collections::page_title_with_date", {#}
+{#    context: context,#}
+{#    date: displayDate#}
+{#  }) }}#}
+{#{% endblock %}#}
+
+{#{% block headerLinks %}{% endblock %}#}
+
+{% block pageContent %}
+
+  {% if resultsAsPopulationTable | length  %}
+  <div class="govuk-grid-row">
+  <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
+      {{ t("dashboard::population.heading") }}
+
+      <span class="govuk-!-display-block">
+          {{ currentWeek | formatDateRangeWithRelativeWeek }}
+        </span>
```

Comment:
Could we align the indentation up on this block:

```nunjucks
{% block pageContent %}
  {% if resultsAsPopulationTable | length  %}
    <div class="govuk-grid-row">
      <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
        <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
          {{ t("dashboard::population.heading") }}

          <span class="govuk-!-display-block">
            {{ currentWeek | formatDateRangeWithRelativeWeek }}
          </span>
        </h2>

        {{ govukTable(resultsAsPopulationTable) }}
      </section>
    </div>
  {% endif %}
{% endblock %}
```

Reply:
Comments removed, and alignment fixed up